wrappers/wrapper.rb

```diff
@@ -1602,6 +1604,10 @@ def simplify_service_setup_duration_and_vehicle_setup_modifiers(vrp, solution =
 
           overall_total_travel_time_correction += total_travel_time_correction
           route.info.total_travel_time -= total_travel_time_correction.round
+
+          # Solution patcher must let the problem untouched
+          vehicle.coef_setup = nil
+          vehicle.additional_setup = nil
```

Comment:
This operation is incorrect I think.

The `:simplify` operation sets these values to 1 and 0
```ruby
vehicle.coef_setup = 1
vehicle.additional_setup = 0
```
and `:rewind` takes these modifications back. Why do we need to set them to `nil` here?